README.md

```diff
@@ -29,14 +38,18 @@ The example uses the [World Cities Dataset](https://simplemaps.com/data/world-ci
   }
 ```
 
-### Database Setup
+## Database setup
 
 It's recommended to use [`Postgres`] for your database so the schema can use `BigInt` and `Float` nicely.
```

Comment:
[sand] Was this meant to be a link to the Postgres website?
```suggestion
It's recommended to use [`Postgres`](https://www.postgresql.org/) for your database so the schema can use `BigInt` and `Float` nicely.
```

Reply:
I don't think so, but I think it's worth the link to it regardless.